src/commands/update-cli.js

```diff
@@ -28,16 +31,22 @@ export default async function () {
 
   if (!updateConfirmed) {
     console.log('Warning: This is an outdated version of the Shoutem CLI.'.bold.yellow);
-    console.log('Install the new one with: `npm install -g @shoutem/cli`. You might need to run it with `sudo` prefix.'.yellow);
+    console.log(`Install the new one with: 'npm install -g @shoutem/cli' or 'yarn global add @shoutem/cli'. You might need to run it with 'sudo' prefix.`.yellow);
     return false;
   }
 
   try {
-    await spawn('npm', ['install', '-g', '@shoutem/cli'], { stdio: 'inherit' });
+    if (packageManager === 'npm') {
```

Comment:
What if the default package manager is yarn?

Reply:
If it's `npm`, it does `npm i -g @shoutem/cli`
Else, it does `yarn global add @shoutem/cli`

Same with sudo.

Currently we only let the user set `npm` or `yarn` as package managers, the `shoutem package-manager` command shows it's usage info if any other package manager is set, which shows the user that `npm` and `yarn` are the only two choices.

If I misunderstood, sorry and please let me know what you meant.

Reply:
Sorry, missed the line after the if block. 

But still I think there's a problem - if package manager is npm, this code is gonna call both `await spawn(packageManager, ['install', '-g', '@shoutem/cli'], { stdio: 'inherit' });` & ` await spawn(packageManager, ['global', 'add', '@shoutem/cli'], { stdio: 'inherit' });` because the second one is non-conditional. 

Am I right?

Reply:
You're absolutely right, that's a mistake on my end. Fixing that right away.